### PR TITLE
setting width for try assistant button

### LINKED ISSUE #3142: Try our assistant button
https://github.com/LAION-AI/Open-Assistant/assets/95025816/9c1ec7eb-4eb5-47c2-96a3-1101df12e6d7

Another one too hard to explain with words...

### PR DESCRIPTION
**Pull request by gsaivinay**
this PR closes #3142

### CODE REVIEW

File: website/src/pages/dashboard.tsx [hunk 1]
```diff
@@ -66,7 +66,7 @@ const Dashboard = () => {
         {ENABLE_CHAT && (
           <Flex direction="column" gap={4}>
             <Heading size="lg">{t("index:try_our_assistant")}</Heading>
-            <Link href="/chat" aria-label="Chat">
+            <Link href="/chat" aria-label="Chat" style={{ width: "max-content" }}>
```
Comment (DoctorKrolic) [inline website/src/pages/dashboard.tsx:69]: Disclamer: I am not a professional frontend developer

Isn't it more correct to put link inside a button? Wouldn't it solve the issue without additional css? Or maybe get rid of button tag at all and make it just link if possible?

Comment (gsaivinay) [reply website/src/pages/dashboard.tsx:69]: Even I thought of same, but that would require refactoring decisions, so I'm not sure what is the process and just raised a PR to fix the existing issue.

Comment (DoctorKrolic) [reply website/src/pages/dashboard.tsx:69]: Well, let's wait for a team member opinion on this then

**Review by notmd — APPROVED** [review]
Thanks

File: website/src/pages/dashboard.tsx (hunk 1 above)
Comment (notmd) [reply website/src/pages/dashboard.tsx:69]: It is better to render the button as a tag, but I think its fine for now.

**Pull request merged**